### PR TITLE
Add Multi spec coerce

### LINKED ISSUE #84: Support multi-spec parsing
see https://github.com/jpmonettas/inspectable/blob/master/src/inspectable/spec_utils.cljc#L24

### PR DESCRIPTION
**Pull request by c0rrzin**
Implements coercion/parsing for multi-specs (Closes #84)

Had some problems with merge, where I had to establish the multi-spec as the last to be merged, so the type can be maintained.

Reviews are appreciated!

With the help of @boschi

### CODE REVIEW

**Comment by ikitommi** [comment]
Looks great! We have a silly coveralls test coverage checker, which says the coverage is less than before, so can't merge this. Could you check if there is something relevant new that could be tested. Thanks.

**Comment by miikka** [comment]
Coveralls is not silly, it is pointing out that the new `(defmethod walk-spec :multi-spec ...)` has not been tested. How about adding a test for that?

File: CHANGELOG.md [hunk 1]
```diff
@@ -1,3 +1,6 @@
+# 0.10.2 (2019-10-31)
```
Comment (miikka) [inline CHANGELOG.md]: Thanks for adding a changelog entry – however, let's add the release number and the date here only once we do the release.

```suggestion
# Unreleased
```

File: project.clj [hunk 2]
```diff
@@ -1,4 +1,4 @@
-(defproject metosin/spec-tools "0.10.1-SNAPSHOT"
+(defproject metosin/spec-tools "0.10.2-SNAPSHOT"
```
Comment (miikka) [inline project.clj]: Please do not update the version number to skip over versions.

```suggestion
(defproject metosin/spec-tools "0.10.1-SNAPSHOT"
```

**Comment by c0rrzin** [comment]
@ikitommi @miikka Thanks for the feedback - I'm implementing your suggestions and this should be enough to keep coverage

**Comment by ikitommi** [comment]
Awesome, thanks!

**Pull request merged**